vizro-core/src/vizro/models/_page.py

```diff
@@ -118,6 +115,11 @@ def pre_build(self):
     def build(self):
         self._update_graph_theme()
         controls_content = [control.build() for control in self.controls]
+        control_panel = (
+            html.Div(children=[*controls_content, html.Hr()], className="control_panel", id="control_panel_outer")
+            if controls_content
+            else html.Div(className="hidden", id="control_panel_outer")
```

Comment:
I see you've changed some of the `None` returns to hidden Divs instead - just curious why is this?

Reply:
It is not always required, but in this case, it is - given that we unpack the page content div later in the `Dashboard` model, the key "control_panel_outer" needs to exist in `page_content`. If we were to return None here, for example,  when there are no controls, they key would not be found and the code wouldn't run. So in this case, you need to create a hidden div that has the id. 

So we currently have the rule of returning hidden divs in case:
- You need to unpack and access the id later
- It's the return object of a build method

I thought it would be confusing to remember this, so I would prefer to always return hidden divs instead of None. WDYT?

Reply:
Ah ok - makes sense in this case, but do we also have a rule for the ones where we don't need to access the `id` later on. Like for `dashboard_title` - we used to leave this as `None` I think but now make a hidden div instead?

I actually think there will be a better way to do all this throughout Vizro which is the "we should discuss hidden `Div`s briefly" thing I mentioned in my previous review, but let's leave it for a separate PR. Happy to go with whatever works here for now.